lobby-db/README.md

```diff
@@ -0,0 +1,94 @@
+## lobby-db
+
+A project that contains the migration scripts for the lobby postgres database.
+
+### Prod Deployments
+- Clone the project to lobby server
+- cd to the `lobby-db` project folder
+
+- run: `./prod_check_flyway.sh`
+  - Follow prompts and add user+password to creds file if not already done.
+
+First time you should see output like this:
+```
+dan@dan-desk:~/work/triplea/lobby-db$ ./prod_run_flyway.sh 
```

Comment:
Probably should remove references to your login.

Reply:
Point taken, this is the example for prod deployment, not a developer deploy. I pushed 441ca4c which would update the example to more 'prod-like.'